runtime.md

```diff
@@ -1,5 +1,51 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information is stored in `/run/oci`.
+The directory structure for a container is `<root>/<containerID>/state.json`.
+
+* **id** (string) ID is the container's ID.
+* **pid** (int) Pid is the ID of the main process within the container.
+* **root** (string) Root is the path to the container's root filesystem specified in the configuration.
+
+*Example*
+
+```json
+{
+    "id": "oci-container",
+    "pid": 4422,
+    "root": "/containers/redis"
+}
+```
+
+Linux systems add some platform specific information to the state.
+
+* **namespaces** Paths to the Linux namespaces setup for the container.
+* **cgroups** Paths to the container's cgroups.
+* **externalFds** Paths to the container's open file descriptors.
```

Comment:
On Wed, Aug 05, 2015 at 03:38:33PM -0700, Brandon Philips wrote:
> > +Linux systems add some platform specific information to the state.
> > +
> > +* **namespaces** Paths to the Linux namespaces setup for the container.
> > +* **cgroups** Paths to the container's cgroups.
> > +* **externalFds** Paths to the container's open file descriptors.
> 
> what is the utility of this? Why can't it be derived from the pid?

+1 to dropping these (previous discussion in [1,2,3,4]).

[1]: https://github.com/opencontainers/specs/pull/87#issuecomment-126119707
[2]: https://github.com/opencontainers/specs/pull/87#issuecomment-126121373
[3]: https://github.com/opencontainers/specs/pull/87#issuecomment-126123937
[4]: https://github.com/opencontainers/specs/pull/87#issuecomment-126124977